utils/DeploymentUtil.js

```diff
@@ -27,6 +27,27 @@ SOFTWARE.
 
 const { UNITS, LOOT, sha3, toBN } = require("./ContractUtil");
 
+const waitForEvent = async (contract, event, times) => {
+  if (!times) times = 1;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819147)
```suggestion
  if (!times) {times = 1;}
```